### DISCUSSION TITLE
Memory address "Outdoor Temperature (est)"

### DISCUSSION THREAD

**Discussion by DIP80** [raomin/ESPAltherma#244] [Q&A]
Hi everybody,please excuse my bad English.
I'm integrating the project with my Daikin EHBX08E heat pump.
in setup.h I selected -> #include "def/ALTHERMA(LT_DA_04-08KW).h"

I used this memory location for the outdoor temperature sensor installed in the outdoor unit. 
0x20,0,105,2,1,"Outdoor air temp.(R1T)

But in the file ALTHERMA(LT_DA_04-08KW).h 
I could not locate the memory location for the "EKRSCA1 Remote outdoor sensor". 
I do not see the value that on the heat pump display is shown as "Outdoor Temperature (est)"

Can you help me?
Thanks

**Comment by KalaNagHTD** [comment]
Try 
{0x61,14,105,2,1,"Ext. indoor ambient sensor (R6T)"}

**Reply by DIP80** [reply]
Thanks, that's right.
I was confused by the description "Indoor".
I think it would be more correct to change the description (and translations) to "Remote outdoor sensor".
Anyway, problem solved,
thanks again.

**Reply by Tardymo** [reply]
That is because R6T can be either indoor or outdoor sensor depending on how you configure your unit.

**Reply by KalaNagHTD** [reply]
Imho the description indicates that it is not the sensor in the outdoor unit but the one which is connected to the indoor unit.

**Reply by Tardymo** [reply]
Better name would be "Ext. ambient sensor (R6T)". It might be that some older models did not have an option for external outdoor sensor, so the register was called "Ext. indoor ambient sensor".

From the manual:

[9.B.1] [C–08]
**External sensor:** When an optional external ambient sensor is connected, the type of the sensor must be set.
- 0 None : NOT installed. The thermistor in the dedicated Human Comfort Interface and in the outdoor unit are used for measurement.
- Outdoor: Connected to PCB of the indoor unit measuring the **outdoor temperature**.
**Remark**: For some functionality, the temperature sensor in the outdoor unit is still used.
- 2 Room: Connected to PCB of the indoor unit measuring the **indoor temperature**. The temperature sensor in the dedicated Human Comfort Interface is NOT used anymore.
**Remark**: This value has only meaning in room thermostat control.
